### DISCUSSION TITLE
Landing and sample coordinates

### DISCUSSION THREAD

**Discussion by pinouche** [esa/SpOC2#12] [Morphing Rovers Q&A] [answered]
I was wondering if the provided coordinates for both landing and samples are the actual ones used in the final competition evaluation?

And if so, this may lead to over-engineered solutions, and perhaps less about learning generalizing solutions?

Thanks!

**Comment by DreamTlaloc** [comment, marked as answer]
Hi Pinouche,

That is indeed correct. We hoped that using a varied set of maps, distances and terrains produces a challenge where the winning solutions are generalists for this problem. Since the rovers will have to face a variety of terrains/distances efficiently to even reach all the samples, and the controller network only takes in a limited input of the area around them, hopefully the best solutions will actually be efficient at understanding the terrain and how to navigate it.

Thanks,
Amy

**Reply by pinouche** [reply]
That makes sense, thanks. Although it may end up giving quite a bit of importance to compute intensive end-to-end brute force.
